inscriptions/ISic0958.xml

```diff
@@ -170,28 +175,28 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140439">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-
-
-                    ἀγορασία<lb n="2"/>Βιταλίου· ἐνθά-<lb n="3"/>δε κῖτε Κυριακὴ<lb n="4"/>ἡ καλῆς μνήμης,<lb n="5"/>
-                    τελευτᾷ δὲ
-                    <supplied reason="omitted">δʹ</supplied>
-                    
-                    μη<surplus>Δ</surplus>νὶ <surplus>²⁶μηνὶ</surplus>²⁶ Μαρτίῳ.
-                    <lb n="6"/>
+                    <lb n="1"/>ἀγορασία
+                    <lb n="2"/>Βιταλίου ἐνθά
+                    <lb n="3" break="no"/>δε κῖτε Κυριακὴ
+                    <lb n="4"/>ἡ καλῆς μνήμης
+                    <lb n="5"/>τελευτᾷ δὲ <choice><corr>μηνὶ</corr><sic>μηδνὶ</sic></choice> Μαρτίῳ
```

Comment:
I wonder if here the delta should be in surplus rather than the whole word in a choice.